Apps/Admin/Client/Services/IConfigurationApi.cs

```diff
@@ -13,22 +13,21 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 //-------------------------------------------------------------------------
-namespace HealthGateway.Admin.Client.Services
-{
-    using System.Threading.Tasks;
-    using HealthGateway.Admin.Common.Models;
-    using Refit;
+namespace HealthGateway.Admin.Client.Services;
+
+using System.Threading.Tasks;
+using HealthGateway.Admin.Common.Models;
+using Refit;
 
+/// <summary>
+/// API to fetch the External Configuration from the server.
+/// </summary>
+public interface IConfigurationApi
+{
     /// <summary>
-    /// API to fetch the External Configuration from the server.
+    /// Gets the configuration from the server for local overrides.
     /// </summary>
-    public interface IConfigurationApi
-    {
-        /// <summary>
-        /// Gets the configuration from the server for local overrides.
-        /// </summary>
-        /// <returns>The ExternalConfiguration object.</returns>
-        [Get("/")]
-        Task<ApiResponse<ExternalConfiguration>> GetConfiguration();
```

Comment:
I don't think we need ApiResponse here as we don't really care why the configuration fails do we? I think in a case like this we should catch the APIException or get the parsed ExternalConfiguration object.